src/components/checkbox.js

```diff
@@ -11,7 +11,7 @@ class MiniMediaPlayerCheckbox extends LitElement {
 
   render() {
     return html`
-      <input type="checkbox" ?checked=${this.checked} ?disabled=${this.disabled}>
+      <input type="checkbox" .checked=${this.checked} ?checked=${this.checked} ?disabled=${this.disabled}>
```

Comment:
Is this needed?
See: [bind-properties-to-templated-elements](https://lit-element.polymer-project.org/guide/templates#bind-properties-to-templated-elements) **Boolean attribute**

Reply:
Actually, it's surprising that it's required, but without that, it doesn't update 🤷‍♂️ I think it's related to how the `input` element works, not related to lit-element (at least that's my understanding).

We need the attribute `?checked` for the css part and the property `.checked` set for the `input` element to update properly.

Reply:
Ok, strange indeed!